docs/core/diagnostics/dumps.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Dumps - .NET
+description: An introduction to dumps in .NET.
+ms.date: 10/12/2020
+---
+
+# Dumps
+
+A dump is a file that contains snapshot of the process at the time they were created and can be useful for examining the state of your application for debugging. Dumps can be used for post-mortem debugging of your .NET application when it is difficult to attach a debugger to it (for example, when the app is running in production). Using dumps allows you to capture the state of the problematic process and examine it without having to stop it from running while being analyzed.
+
+## Collect dumps
+
+Dumps can be collected in a variety of ways depending on which platform you are running your app on.
+
+> [!NOTE]
+> Collecting a dump inside a container requires PTRACE capability which can be added via `--cap-add=SYS_PTRACE` or `--privileged`.
+
+### Using dotnet-dump
+
+`dotnet-dump` is a CLI tool that allows you to collect and analyze dumps. For more information on how to use it to collect dumps with `dotnet-dump`, refer to [its documentation](dotnet-dump.md)
+
+### Using environment variables
+
+For some situations, it may be more desirable to use environment variables to collect dumps. For example, capturing a dump when an exception is thrown helps you identify issue by examining the state of the app when it crashed.
+
+The table below shows the environment variables you can configure for collecting dumps.
+
+|Environment variable|Description|Default value|
+|-------|---------|---|
+|`COMPlus_DbgEnableMiniDump`|If set to 1, enable core dump generation.|0|
+|`COMPlus_DbgMiniDumpType`|Type of the dump to be collected. (See [Types of dumps](#types-of-dumps-in-.net) to learn more)|2 (`MiniDumpWithPrivateReadWriteMemory`)|
+|`COMPlus_DbgMiniDumpName`|Path to a file to write the dump to.|`/tmp/coredump.<pid>`|
+|`COMPlus_CreateDumpDiagnostics`|If set to 1, enable diagnostic logging of dump process.|0|
+
+### Types of dumps in .NET
```

Comment:
Conceptually this is a subtitle of the last bullet. It could have an extra level of the header, but i'd say it's enough to say "where COMPlus_DbgMiniDumpType is one of the following"

Reply:
I wanted to write it as a separate bullet because they can be used with `dotnet-dump` as well. Does that sound fine to you?

Reply:
They don't map one to one, and the dotnet-dump article has it, with it's own blurb. If we are going to point them there, might as well not randomize them in this article anymore.